.vscode/launch.json

```diff
@@ -0,0 +1,36 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Is this file part of our source code as it is under .vscode folder or can this be ignored?

Reply:
It's the configuration file for debugging. Not technically part of the code, but useful for anyone who wants to debug, so I left it in. It's similar to the `.idea` foler that Intellj creates